.github/workflows/release-published.yml

```diff
@@ -0,0 +1,22 @@
+name: Checks after release is published
+on:
+  release:
+    types: ['published']
+
+jobs:
+  validate-github-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: k14s/setup-k14s-action@v1
+        with:
+          only: ytt
+      - run: |
+          ytt version
+          version=`ytt version`
+          release_version="${GITHUB_REF:1}"
+          if [[ "${version}" == *"$release_version"* ]];
```

Comment:
My own uncertainty: are *s wildcarding? 

If so, what's the reason behind not expecting an exact match?